web/src/main/java/org/fao/geonet/proxy/URITemplateProxyServlet.java

```diff
@@ -116,6 +141,65 @@ protected void initTarget() throws ServletException {
                 doForwadHostPrefixPathString != null ? doForwadHostPrefixPathString : "";
         }
         super.initTarget();
+
+        // If baseUri is not set in web.xml try to resolve it from java properties or environment variables.
```

Comment:
Maybe we should do the other way around - use any external configuration if defined and if not fallback to the web.xml value ?

Reply:
It could be useful to override ESFeaturesProxy targetUri too which is not empty by default.

Reply:
Updated to use the property from `web.xml` in last position.